### PR TITLE
5.x: fix plugins.php not loading in consecutive tests due to include_once

### PR DESCRIPTION
**Pull request by LordSimal**
As can be seen in https://github.com/cakephp/app/pull/971 there is a problem with the current implementation of how `config/plugins.php` is loaded.

`include_once` will only return the contents of that file in the first running test.
But if another test tries load plugins again this doesn't work any more due to how `include_once` works.

### CODE REVIEW

**Comment by LordSimal** [comment]
I am very confused why this change triggers this error...

**Comment by ndm2** [comment]
It's because `plugin load` adds the plugin to the test app's `config/plugins.php` file accordingly, and each `plugin load` command executed in the test will also bootstrap the test app again, with that modified plugin config file, causing previously loaded plugins to be loaded again, which will then cause previously loaded routes files to be loaded again, as they're included using `require`.

tl;dr call `Router::reload()` before `$this->exec()`.

**Comment by LordSimal** [comment]
Thanks ndm for clarifying that 👍🏻
